event/datadeal/data_verification_event.go

```diff
@@ -53,61 +45,61 @@ func (d DataVerificationEvent) EventHandler(event ctypes.ResultEvent) error {
 		return err
 	}
 
-	voteOption, err := d.verifyAndGetVoteOption(dealID, dataHash)
+	voteOption, err := e.verifyAndGetVoteOption(dealID, dataHash)
 	if err != nil {
-		log.Infof("can't vote due to error while verify. dealID(%d). dataHash(%s)", dealID, dataHash)
+		log.Infof("vote No due to error while verify. dealID(%d). dataHash(%s)", dealID, dataHash)
 	}
 
 	msgVoteDataVerification, err := makeDataVerificationVote(
-		d.reactor.OracleAcc().GetAddress(),
+		e.reactor.OracleAcc().GetAddress(),
 		dataHash,
 		dealID,
 		voteOption,
-		d.reactor.OraclePrivKey().Serialize(),
+		e.reactor.OraclePrivKey().Serialize(),
 	)
 	if err != nil {
 		return err
 	}
 
-	txBuilder := panacea.NewTxBuilder(*d.reactor.QueryClient())
+	txBuilder := panacea.NewTxBuilder(*e.reactor.QueryClient())
 
-	txBytes, err := generateTxBytes(msgVoteDataVerification, d.reactor.OracleAcc().GetPrivKey(), d.reactor.Config(), txBuilder)
+	txBytes, err := generateTxBytes(msgVoteDataVerification, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), txBuilder)
 	if err != nil {
 		return fmt.Errorf("generate tx failed. dealID(%d). dataHash(%s): %w", dealID, dataHash, err)
 	}
 
-	if err := broadcastTx(d.reactor.GRPCClient(), txBytes); err != nil {
+	if err := broadcastTx(e.reactor.GRPCClient(), txBytes); err != nil {
 		return fmt.Errorf("broadcast transaction failed. dealID(%d). dataHash(%s): %w", dealID, dataHash, err)
 	}
 
 	return nil
 }
 
-func (d DataVerificationEvent) decryptData(decryptedSharedKey, nonce, encryptedDataBz []byte) ([]byte, error) {
+func (e DataVerificationEvent) decryptData(decryptedSharedKey, nonce, encryptedDataBz []byte) ([]byte, error) {
 	decryptedData, err := crypto.DecryptWithAES256(decryptedSharedKey, nonce, encryptedDataBz)
 	if err != nil {
 		return nil, err
 	}
 	return decryptedData, nil
```

Comment:
How about changing as direct return?
```suggestion
	return crypto.DecryptWithAES256(decryptedSharedKey, nonce, encryptedDataBz)
```

Reply:
Thanks, I reflected this in [dc45af6](https://github.com/medibloc/panacea-doracle/pull/95/commits/dc45af634288b9de9e1ce6b42d75b77bbeaf3625). 👍